lib/DxilValidation/DxilValidation.cpp

```diff
@@ -1955,6 +2171,16 @@ static void ValidateDxilOperationCallInProfile(CallInst *CI,
                                  GetLaunchTypeStr(NodeLaunchType)});
 
     break;
+  case DXIL::OpCode::MatVecMul:
+  case DXIL::OpCode::MatVecMulAdd:
+    ValidateImmOperandsForMatVecOps(CI, Opcode, ValCtx);
+    break;
+  case DXIL::OpCode::OuterProductAccumulate:
+    ValidateImmOperandsForOuterProdAcc(CI, ValCtx);
+    break;
+  case DXIL::OpCode::VectorAccumulate:
+
```

Comment:
Is it TBD to validate the resource handle type, or that required dimensions are consistent with argument values, or anything else?

One thing I noticed is that the spec doesn't have any details under the `Validation Changes` section, but does have points for validating just the input/matrix/bias interpretation args for each op.  It doesn't include some additional argument validation already implemented here.

Is there a spec update coming to fill out more validation plans?  Is there an issue tracking this?

Reply:
Created an issue: https://github.com/microsoft/hlsl-specs/issues/491
Let me know if you had any specific ones in mind. I'll also go through the spec and update the issue with more specifics.